doc/releases/reef.rst

```diff
@@ -11,12 +11,14 @@ This is the seventh backport (hotfix) release in the Reef series. We recommend t
 Notable Changes
 ---------------
 
-This release fixes a critical bluestore regression in versions 18.2.5 and 18.2.6 (https://github.com/ceph/ceph/pull/61653). Users running either of those releases are recommended to upgrade at the earliest convenience.
+This release fixes a critical bluestore regression in versions 18.2.5 and 18.2.6 (https://github.com/ceph/ceph/pull/61653).
```

Comment:
BlueStore

Also suggest, though it may seem obvious, that nobody install new clusters with the problematic releases.

Reply:
^capitalization